src/caffeine/image_enumeration_m.f90

```diff
@@ -1,18 +1,18 @@
 ! Copyright (c), The Regents of the University of California
 ! Terms of use are as specified in LICENSE.txt
 module image_enumeration_m
-  use team_type_m, only : team_type
+  use team_type_m, only : prif_team_type
   implicit none
 
   private
-  public :: caf_num_images
-  public :: caf_this_image
+  public :: prif_num_images
+  public :: prif_this_image
 
-  interface caf_num_images
+  interface prif_num_images
```

Comment:
`prif_num_images` is currently specified as one function with two optional arguments.